app/org/sagebionetworks/bridge/cache/CacheProvider.java

```diff
@@ -89,7 +93,42 @@ public Study getStudy(String identifier) {
     public void removeStudy(String identifier) {
         try {
             String redisKey = RedisKey.STUDY.getRedisKey(identifier);
-            stringOps.delete(redisKey).execute();
+            stringOps.delete(redisKey);
+        } catch(Throwable e) {
+            promptToStartRedisIfLocalEnv(e);
+            throw new BridgeServiceException(e);
+        }
+    }
+    
+    public String getString(String cacheKey) {
+        try {
+            String ser = stringOps.get(cacheKey);
+            if (ser != null) {
+                stringOps.expire(cacheKey, BridgeConstants.BRIDGE_SESSION_EXPIRE_IN_SECONDS);
```

Comment:
I think this is okay if we're manually expiring the cache (it could never expire). I will change the expire constant if we keep it.